.versions/ide-sidecar.txt

```diff
@@ -1 +1 @@
-v0.51.0
+v0.53.0
```

Comment:
Setting back to draft until https://github.com/confluentinc/ide-sidecar/pull/90 merges and we can bump the sidecar version one more time